client/packages/invoices/src/InboundShipment/DetailView/AppBarButtons.tsx

```diff
@@ -31,6 +34,11 @@ export const AppBarButtonsComponent: FC<AppBarButtonProps> = ({
   const { OpenButton } = useDetailPanel();
   const t = useTranslation('common');
   const { print, isPrinting } = useReport.utils.print();
+  const pluginButtons = usePluginElements({
+    area: PluginArea.AppBar,
```

Comment:
I believe this is Toolbar? The provided plugin then loads but crashes... Not sure if I have an outdated plugin though...